tools-overview.md

```diff
@@ -0,0 +1,349 @@
+
+```shell
+luis init
+```
+You will be prompted in the terminal to enter your LUIS authoring key, region, and appID before the tool will generate the file.  
+
+Once this file is generated, your application will be able to consume the LUIS .json file (generated from LUDown) using the following command from the CLI.
+
+```shell
+luis import application --in luis-app.json | msbot connect luis --stdin
+```
+See [here](https://github.com/Microsoft/botbuilder-tools/tree/master/packages/LUIS) to learn more about the LUIS CLI.
+
+## Create QnA Maker KB using QnA Maker CLI
+
+Included in the new tool set is a [QnA extension](https://github.com/Microsoft/botbuilder-tools/tree/master/packages/QnAMaker) which allows you to independently manage your LUIS resources. It is available as an npm module which you can download.
+
+```shell
+npm install -g qnamaker
+```
+With the QnA maker tool, you can create, update, publish, delete, and train your knowledge base. You can use files generated via [ludown parse toqna](#generate-qna-knowledge-base) command to create/ replace a knowledge base.
+
+```shell
+qnamaker create --in qnaKB.json --msbot | msbot connect qna --stdin
+```
+
+See [here](https://github.com/Microsoft/botbuilder-tools/tree/master/packages/QnAMaker) to learn more about the QnA Maker CLI.
+
+## Create dipsatch model using Dispatch CLI
+
+Dispatch is a tool to create and evaluate LUIS models used to dispatch intent across multiple bot modules such as LUIS models, QnA knowledge bases and others (added to dispatch as a file type).
+
+Use the Dispatch model in cases when:
+
+- Your bot consists of multiple modules and you need assistance in routing user's utterances to these modules and evaluate the bot integration.
+- Evaluate quality of intents classification of a single LUIS model.
+- Create a text classification model from text files.
+
+Once you have assembled your .bot file with the [LUIS applications](https://github.com/Microsoft/botbuilder-tools/tree/master/packages/MSBot/docs/msbot-luis.md) and [QnA Maker knowledge bases](https://github.com/Microsoft/botbuilder-tools/tree/master/packages/MSBot/docs/msbot-qnamaker.md) your bot relies on, you can simply build a dispatch model using: 
+
+```shell
+dispatch create -b <YOUR-BOT-FILE> | msbot connect dispatch --stdin
+```
+See [here](https://github.com/Microsoft/botbuilder-tools/tree/master/packages/Dispatch) to learn more about the Dispatch CLI.
+
+# Test
+
+The Bot Framework Emulator is a desktop application that allows bot developers to test and debug their bots on localhost or running remotely through a tunnel.
+
+## Download
+
+* From the [GitHub releases](https://github.com/Microsoft/BotFramework-Emulator/releases) page
+* From the [Bot Framework](https://emulator.botframework.com) site
```

Comment:
Do we need to provide two links for the same download? It might confuse the user.